rio/src/RIO/Prelude/Exit.hs

```diff
@@ -0,0 +1,15 @@
+module RIO.Prelude.Exit
+  ( module System.Exit
+  , exitSuccess
+  , exitFailure
+  ) where
+
+import           Control.Monad.IO.Class
+import           System.Exit hiding( exitFailure, exitSuccess )
+import qualified System.Exit
+
+exitFailure :: MonadIO m => m a
+exitFailure = liftIO System.Exit.exitFailure
+
+exitSuccess :: MonadIO m => m a
+exitSuccess = liftIO System.Exit.exitSuccess
```

Comment:
Should we also include `exitWith`? And I think `ExitCode` is already reexported elsewhere, but perhaps for logical consistency it should be reexported here.

Reply:
I'd agree, done!